frontend/src/constants/routes.js

```diff
@@ -11,6 +11,7 @@ export const EVENT_EDIT = '/events/:eventId/edit';
 export const EVENT_DETAILS = '/events/:id';
 export const EVENT_CREATION = '/events?create=true';
 export const TEST = '/test';
+export const EDIT_ROLES = '/editroles';

```

Comment:
```suggestion
export const EDIT_ROLES = '/roles';
```
in general routes don't have verbs